lab-zach/lib/greet.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = function(arg) {
+  if (arg) {
```

Comment:
Part of the lab requirements was to validate that the input was a string (rather than a Boolen, number, etc.) Add it for the last two points.